sync/test_sync.py

```diff
@@ -120,44 +119,20 @@ def test_get_files(self):
         expected = None
         actual = None
 
-        with tempfile.NamedTemporaryFile(delete=True) as tmp:
+        with tempfile.NamedTemporaryFile(dir='/tmp', delete=True) as tmp:
```

Comment:
Assuming `/tmp` is the temp folder is not portable, so I set the base dir explicitly to make the test run on mac.